renovate.json

```diff
@@ -1,6 +1,13 @@
 {
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
-  "extends": ["config:base"],
-  "enabledManagers": ["npm"],
-  "rangeStrategy": "update-lockfile"
+  "extends": [
+    "config:base"
+  ],
+  "enabledManagers": [
+    "npm"
+  ],
+  "rangeStrategy": "update-lockfile",
+  "schedule": [
+    "every 3 months on the first day of the month"
```

Comment:
I agree with the noise issue. Especially in this repository, where (At least to me) testing if anything breaks is a hassle. But my vote would be on seeing if once a month would work. I could even take the opportunity to take a more active role in the approval pipeline for renovate.

Reply:
it is done!